clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractStickyAssignorTest.java

```diff
@@ -425,8 +422,36 @@ public void testSameSubscriptions() {
         assertTrue(assignor.isSticky());
     }
 
+    @Test(timeout = 30 * 1000)
+    public void testLargeAssignmentAndGroupWithUniformSubscription() {
+        int topicCount = 200;
```

Comment:
On trunk, this test fails (hits the 30s timeout) even when you reduce the number of topics to just 1